lessons/linear-regression/langs/en-us/pages/lr-plot-intro.adoc

```diff
@@ -0,0 +1,55 @@
+= Introduction to Linear Regression
+
+*Part 1: Predictions from Scatter Plots?*
+
+[cols="1,1", frame="none", grid="none", stripes="none"]
+|===
+| @centered-image{../images/age-v-weeks.gif, scatter plot age v weeks} | @centered-image{../images/pounds-v-weeks.gif, scatter plot pounds v weeks}
+|===
+
+@n What do you think: Can we predict the number of weeks til adoption from an animal's weight?
```

Comment:
Minor nit -- since the first scatter plot is about age, can the first question also be about age?